jeos-firstboot-enroll

```diff
@@ -133,7 +127,8 @@ enroll_post() {
 	for dev in "${crypt_devs[@]}"; do
 	    enroll_tpm2 "$dev"
 	done
-	crypttab_options+=",tpm2-device=auto"
+	ct_pcrs="${pcrs//,/+}"
+	crypttab_options+=",tpm2-device=auto,tpm2-pcrs=$ct_pcrs"
```

Comment:
I don't think this has an effect. We're using the token for that

Reply:
Can be, but an issue that I saw is that the luks token can report something in the metadata (like it is using pcr 7, the default one) but in reality it is using a signed policy and can be different in each situation (and in fact is using a different pcr combination)

Seems to me that the luks token can be totally out of sync and we need a proper reference. Some alteratives are:

* a config file (I personally do not like this one)
* the entry in crypttab (I like it more, but is signaling something different: that the luks can be unlocked with a PCR policy instead of an authorization policy)
* some EFI variable (not sure we can trust it)
* parsing the previous JSON file (as commented, the PCRs can be different for each entry)
* something else

Reply:
I will recheck, maybe the field tpm2-pubkey-pcrs is more correct

Reply:
Moved to tokens